client/components/theme/more-button.jsx

```diff
@@ -20,15 +20,33 @@ var PopoverMenu = require( 'components/popover/menu' ),
  */
 var ThemeMoreButton = React.createClass( {
 	propTypes: {
-		// Theme ID (theme-slug)
-		id: React.PropTypes.string.isRequired,
-		// Index of theme in results list, if any
+		theme: React.PropTypes.shape( {
```

Comment:
Interesting, this PropType is too complex to repeat in various places. Should we factor out this PropType somewhere so it can be shared? Might be better just to have it in one place and expect a PropType of object elsewhere.